www/src/app/components/inbox/Inbox.js

```diff
@@ -1,122 +1,126 @@
+            <UpdateList
+              className="tile tile--scratchpad"
+              quips={genericMessages}
+              type={'CurrentGoal'}
+              name="Your current goals"
+              listItems={this.props.prevgoals}
+            />
+
+            <UpdateList
+              className="tile tile--scratchpad"
+              quips={genericMessages}
+              type={'Todo'}
+              name="To do"
+              listItems={this.props.todo}
+            />
+
+            <UpdateList
+              className="tile tile--scratchpad"
+              quips={genericMessages}
+              type={'Done'}
+              name="Done"
+              listItems={this.props.done}
+            />
+          </div>
 
+          <div className="content__column">
+            <section className="tile tile--report">
+              <h2 className="tile__title tile__title--report">
+                Report for {reportName(nextReportDate)}
+                <div className="tile__actions">
+                  <Link className="action" to={`/report/${nextReportSlug}`}>see full report</Link>
+                  <WikiTextLink {...this.props} className="action">wikify</WikiTextLink>
+                </div>
+              </h2>
+              <p className="tile__hint">Prepare the report for the next weekly meeting.</p>
+              <h3 className="tile__subtitle">
+                Goals for next week
+                <div className="tile__actions">
+                  <button className="action" onClick={() => this.props.handleStartAdd('goal')}>add</button>
+                </div>
+              </h3>
+              <ul className="tile__updates">
+                {this.props.goals.map(update => createUpdate(Reviewed, this.props, update))}
+              </ul>
+
+              <h3 className="tile__subtitle">
+                Struggles this week
+                <div className="tile__actions">
+                  <button className="action" onClick={() => this.props.handleStartAdd('struggle')}>add</button>
+                </div>
+              </h3>
+              <ul className="tile__updates">
+                {this.props.struggles.map(update => createUpdate(Reviewed, this.props, update))}
+              </ul>
+
+              <h3 className="tile__subtitle">
+                Achievements this week
+                <div className="tile__actions">
+                  <button className="action" onClick={() => this.props.handleStartAdd('achievement')}>add</button>
+                </div>
+              </h3>
+              <ul className="tile__updates">
+                {this.props.achievements.map(update => createUpdate(Reviewed, this.props, update))}
+              </ul>
+            </section>
+          </div>
+
+        </div>
       </div>
-    </div>
-  );
+    );
+  }
 }
+
+const mapStateToProps = (state) => ({
+  author: state.inbox.author,
+  nextReportDate: state.inbox.nextReportDate,
+  nextReportSlug: state.inbox.nextReportSlug
```

Comment:
ok, since I'm not setting that in a reducer yet - (I wanted to contain this to the initial `fetchCurrentReports` and `fetchUpdatesByAuthor` methods, neither of which seems to return the `prevReportSlug` - I think we can make sure to add that in a future PR.